pages/editor/[pid].tsx

```diff
@@ -70,25 +71,30 @@ const UpdateArticleEditor = ({ article: initialArticle }) => {
       initialState.description="This article has no description"
     }
     initialState.body=value_dummy
-    setLoading(true);
-    const { data, status } = await axios.put(
-      `${SERVER_BASE_URL}/articles/${pid}`,
-      JSON.stringify({ article: initialState }),
-      {
-        headers: {
-          "Content-Type": "application/json",
-          Authorization: `Token ${encodeURIComponent(currentUser?.token)}`,
-        },
+    if(title!=""){
+      setLoading(true);
+      const { data, status } = await axios.put(
+        `${SERVER_BASE_URL}/articles/${pid}`,
```

Comment:
Each time that the user types in markdown editor and the article has a title, it should call the `/api/articles/{{slug}}/draft` route to save the article.

Reply:
changed the call to `${SERVER_BASE_URL}/articles/${pid}/draft`

Reply:
Ok so I realized that I was being dumb. We could just use the `/api/articles/<slug>` PUT route to continuously update the article.